### ISSUE TITLE
Add a new field storageLocation in the Container.

### ISSUE THREAD

**Issue by delageniere** [ispyb/ispyb-database-documentation#47]
Allow the location of individual containers to be tracked on site.
Default value should be set to DewarStorageLocation value, but can be changed if needed.

**Comment by stufisher** [comment]
There is already ContainerHistory which could be used for this if you join the last value?

**Comment by stufisher** [comment]
It would be nice to do the same for dewar to be honest, drop dewarstoragelocation and use only DewarHistory, saves having to keep two things synchronised manually

**Comment by delageniere** [comment]
> 
> 
> There is already ContainerHistory which could be used for this if you join the last value?

Yes, this is a good point and this will allow to have the history, not sure it is needed for now, but we could use this.

**Comment by delageniere** [comment]
> 
> 
> It would be nice to do the same for dewar to be honest, drop dewarstoragelocation and use only DewarHistory, saves having to keep two things synchronised manually.

It's true that only one is needed, but in the DewarLocation we also have the userId of the person who locates the dewar. We have to check if this is needed.
